testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import xarray as xr
+import matplotlib
```

Comment:
Might want to switch to the `Agg` backend here so it works on compute nodes, too, and without x-forwarding.